modules/ROOT/pages/index-secrets-manager.adoc

```diff
@@ -5,13 +5,52 @@ endif::[]
 
 Secrets Manager uses secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.
```

Comment:
```suggestion
The secrets manager uses secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.
```
Per PMMs, "secrets manager" is lowercase because it is "capability terminology, no a product": https://docs.google.com/spreadsheets/d/1f6v9cjx84ZAHGeZs36A0clZoQc_ggAe6m6e-NQ9BUuk/edit#gid=71218867

You can pursue this on the Slack channel product-naming-questions, or I can do that for you.

GLOBAL; I will stop marking this.